catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -836,4 +846,28 @@ private static void assertDateAttribute(
       assertDateAttribute(reason, attribute, expectedDateTimes);
     }
   }
+
+  public static class NitfValue {
+    private Object value;
+
+    private Object extValue;
+
+    public NitfValue(Object value, Object extValue) {
+      this.value = value;
+      this.extValue = extValue;
+    }
+
+    public NitfValue(Object value) {
+      this.value = value;
+      this.extValue = value;
+    }
+
+    public Object getValue() {
```

Comment:
:pencil2: I think we should name this more descriptively.  Maybe `getTaxonomyValue` or `getNormalizedValue`.

Reply:
I agree. Fixed.